projections/gorm.go

```diff
@@ -99,7 +103,7 @@ func (p *GORMProjection) Remove(entities []weos.Entity) error {
 	return nil
 }
 
-func (p *GORMProjection) Migrate(ctx context.Context, builders map[string]ds.Builder) error {
+func (p *GORMProjection) Migrate(ctx context.Context, builders map[string]ds.Builder, refs map[string]*openapi3.SchemaRef) error {
```

Comment:
it would be clearer to have a property that was a map of the table name and fields to remove that pass the schema ref.